Documentation/gettingstarted/k8s-install-connectivity-test.rst

```diff
@@ -15,7 +15,7 @@ indicates success or failure of the test:
 
 .. code:: shell-session
 
-   $ kubectl get pods -n cilium-test
+   $ kubectl get pods
```

Comment:
The [troubleshooting guide](https://docs.cilium.io/en/latest/operations/troubleshooting/#cilium-connectivity-tests) recommends deploying the connectivity test into a `cilium-test` namespace. I think we should aim for consistency. What about adding a step above to `kubectl create ns cilium-test` instead?

Reply:
SGTM, I'll update the this PR to include that.

Reply:
PR updated.